ctl/src/cli.rs

```diff
@@ -0,0 +1,168 @@
+#[derive(StructOpt, PartialEq, Debug)]
+pub struct App {
+  #[structopt(short="c", long = "config", help = "Sets a custom config file")]
+  pub config: String,
+  #[structopt(subcommand)]
+  pub cmd: SubCmd,
+}
+
+#[derive(StructOpt, PartialEq, Debug)]
+pub enum SubCmd {
+  #[structopt(name = "shutdown", about = "shuts down the proxy without waiting for connections to finish")]
+  Shutdown {
+    #[structopt(short = "h", long = "hard")]
+    hard: Option<bool>
```

Comment:
instead of an `Option<bool>`, use `default_value = false` (is it possible to set a boolean there?)